### ISSUE TITLE
app-timefiller does not work with personium-core v1.7.18b

### ISSUE THREAD

**Issue by tochi-y** [personium/app-timefiller#40]
This error message is displayed in the screen.

<img src="https://user-images.githubusercontent.com/32325140/66016642-6ea52700-e512-11e9-9d5b-eb7c3259cc43.png" width="320px" />

### http access

**request**
POST https://tochi.test.example/__token
Requestbody:
```
grant_type: refresh_token
refresh_token: RR~v9Jy3*************Eow
client_id: https://app-timefiller-wakaba.tochi.test.example/
client_secret: PEFz***************b24-
```

**response**
Status Code: 401 Unauthorized
Responsebody:
```json
{
  "error_description": "[PR401-AN-0020] - Client mismatch for refresh token. [https://app-timefiller-wakaba.text.example/]",
  "error": "invalid_client"
}
```

**Comment by shimono** [comment]
This error is returned when switching applications upon refreshing tokens.

**Comment by tochi-y** [comment]
As well as personium/app-personium-calendar#155, I created a new environment with v1.7.19 and did a same test. Same issue is completely occurred.

### Request

GET https://tochi.test.example/timefiller/
Authorization: Bearer AV~*********E5A

### Response

Status Code: 403
Body:

```json
{
  "code": "PR403-AU-0002",
  "message": {
    "lang": "en",
    "value": "Necessary privilege is lacking."
  }
}
```

@shimono Is it a personium-core bug?

**Comment by tochi-y** [comment]
Resolved in personium-core v1.7.20. But there is an another issue #41.
